extensions/cli/osm/src/main/java/mil/nga/giat/geowave/cli/osm/operations/IngestOSMToGeoWaveCommand.java

```diff
@@ -121,11 +81,13 @@ private void ingestData()
 					"OSMRunner failed: " + res);
 		}
 
-		System.out.println("finished ingest");
-		System.out.println("**************************************************");
+		List<String> output = new ArrayList<>();
+		output.add("finished ingest");
+		output.add("**************************************************");
```

Comment:
its pretty silly to add this these `*`'s to the service endpoint responses - the * is really CLI print outs only